.vscodeignore

```diff
@@ -11,5 +11,4 @@ src/**
 vscodeNim/**
 .gitignore
 tsconfig.json
-tslint.json
-node_modules/**
```

Comment:
Why remove this and are the line endings incorrect?

Reply:
the vsix has to package `"vscode-languageclient": "7.0.0"`

Reply:
It's been a while since I looked at vscodeignore, that'll preclude is from including the client?

Reply:
Yes, `vsce` looks into this file and based on it includes or does not a particular file/directory.

Reply:
Got it, can we restrict it so it only looks at that module, node modules is huge